src/wp-includes/formatting.php

```diff
@@ -5394,6 +5394,28 @@ function normalize_whitespace( $str ) {
  * @return string The processed string.
  */
 function wp_strip_all_tags( $text, $remove_breaks = false ) {
+	if ( is_null( $text ) ) {
+		return '';
+	}
+
+	if ( ! is_scalar( $text ) ) {
+		if ( WP_DEBUG ) {
+			trigger_error(
+				sprintf(
+					/* translators: 1: The function name, 2: The argument number, 3: The expected type, 4: The provided type. */
+					__( 'Warning: %1$s expects parameter %2$d to be a %3$s, %4$s given.' ),
+					__FUNCTION__,
+					'1',
```

Comment:
```suggestion
					/* translators: 1: The function name, 2: The argument number, 3: The argument name, 4: The expected type, 5: The provided type. */
					__( 'Warning: %1$s expects parameter %2$d (%3$s) to be a %4$s, %5$s given.' ),
					__FUNCTION__,
					'#1',
					'$text',
```

@jrfnl Is this what you were thinking, it matches https://3v4l.org/v6sNt but that may be a custom config.

Reply:
Something along these lines and yes, matching the PHP native messages closely is never a bad thing, so happy to follow that. (AFAIK 3v4l doesn't have any custom config for things like this, you see what PHP shows)

Reply:
Updated in [0087427](https://github.com/WordPress/wordpress-develop/pull/3132/commits/0087427ba4a338edeec9bf0c58ab2386e0eea256).